Source/Tools/run_diff.py

```diff
@@ -135,6 +149,47 @@ def post_github_pr_text_comment(text, pr_number, token):
     return response.status_code
 
 
+def post_github_pr_file_scoped_comment_with_images(file_id, directory_with_images, pr_number, token, commit_id):
+    # First, upload all the images.  Since PRs do not support assets, this code uploads images
```

Comment:
This is unfortunate. I did some googling, and while I found some alternative solutions, none of them are obviously better than what you have here.